### ISSUE TITLE
Psp java 

### ISSUE THREAD

**Issue by skelletR** [nikita36078/J2ME-Loader#313]
**Emulator version:**1.2.8.8-play 

**Game version:**

**Game resolution:**
``(For example, 240x320 or 640x360)``

**Device:**
``(Sony Xperia Z5 )``

**Android version:**7.1.1

**Description of the issue:**
``In my psp I have the emulator PSPKVM for playing java games and I have lot of save but in a db file I try to copy as a text and paste in the rsr file and change name but not working ``

**Comment by AdityaMH** [comment]
That's not emulator bug. because if copied as text, some unknown text (such as hex) will be missing or uncopied.

**Comment by skelletR** [comment]
Yes I know not bug.. Just ask from curious if can do this

**Comment by AdityaMH** [comment]
each emulator have different saving data game method (with/out header inside file for example). i think that's uncertainly working expect they have same plugin/method.

**Comment by skelletR** [comment]
Yes I know.. I try.. Because have many save in another emulator in my psp but  Unsuccesful

**Comment by jasper1412** [comment]
@skelletR pspkvm differs from j2me-loader at some point in emulation, so we should not expect variable x to work with variable y. I appreciate your curiosity though.

**Comment by skelletR** [comment]
Yes I know very different.. But it's ok understand
